api/data_refinery_api/test/test_api_general.py

```diff
@@ -286,7 +286,7 @@ def test_all_endpoints(self):
 
         response = self.client.get(
             reverse("transcriptome_indices", kwargs={"version": API_VERSION})
-            + "?organism__name=DANIO_RERIO"
+            + "?organism_name=DANIO_RERIO"
```

Comment:
I think that extra underscore was intentional, it denotes that `name` is a subfield of `organism`.

Reply:
ok it looks like it'll need to be changed in `trancriptome_indices` then

Reply:
I think that's correct. There's no `organism_name` field on the OrganismIndex model.